RecoEcal/EgammaClusterAlgos/src/EndcapPiZeroDiscriminatorAlgo.cc

```diff
@@ -10,52 +10,30 @@
 
 using namespace std;
 
+namespace {
+  constexpr int Nfiles_EE = 5;
+}
+
 EndcapPiZeroDiscriminatorAlgo::EndcapPiZeroDiscriminatorAlgo(double stripEnergyCut, int nStripCut, const string& path) :
-   preshStripEnergyCut_(stripEnergyCut),  preshSeededNstr_(nStripCut), pathToFiles_(path)
+   preshStripEnergyCut_(stripEnergyCut),  preshSeededNstr_(nStripCut)
 {   
    
      // Read all Weight files
-     Nfiles_EB = 5;
-     Nfiles_EE = 5;
+     const int Nfiles_EB = 5;
```

Comment:
Why do you treat differently ```Nfiles_EE``` and ```Nfiles_EB```?

Reply:
Nfiles_EE was being used in two different member functions while Nfiles_EB was used in only one.